components/nimbus/src/dbcache.rs

```diff
@@ -98,9 +110,19 @@ impl DatabaseCache {
     }
 
     pub fn get_experiment_branch(&self, id: &str) -> Result<Option<String>> {
-        self.get_data(|data| match data.branches_by_feature.get(id) {
+        self.get_data(|data| match data.features_by_feature_id.get(id) {
             None => data.branches_by_experiment.get(id).cloned(),
-            Some(branch_slug) => Some(branch_slug.to_owned()),
+            Some(feature) => Some(feature.branch.clone()),
+        })
+    }
+
+    pub fn get_feature_config_variables(&self, feature_id: &str) -> Result<Option<String>> {
```

Comment:
This will change to the `EnrolledExperiment` or the `FeatureConfig.value` when uniffi gets these capabilities.

Reply:
I'm sorry, I meant `EnrolledFeature`. Although now it won't.

We may consider a further struct to mimic `EnrolledFeature` to go over the FFI with the experiment slug in it.

Reply:
I can't find an `EnrolledFeature`.  Perhaps `EnrolledFeatureConfig` is what you mean?

I'm assuming you're talking about at some future time, and not as part of this PR.  Correct?

Reply:
Yes.